<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>541</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export ASCII</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="output4thColumnFlag">
       <property name="text">
        <string>with 4th column (precision)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="auto_qmin_button">
          <property name="text">
           <string>Auto Qmin</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="manual_qmin_frame">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Qmin</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="manual_qmin_value">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>27</height>
              </size>
             </property>
             <property name="text">
              <string>0.005</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>1/Å</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Resolution</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;dQ&lt;span style=&quot; vertical-align:sub;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dq0Value">
            <property name="text">
             <string>0.0004</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>1/Å</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;ΔQ&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;/Q&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dQoverQvalue">
            <property name="text">
             <string>0.005</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>  Output format</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="one_ascii_format">
          <property name="text">
           <string>1 Ascii File for All Reduced Data Set</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="n_ascii_format">
          <property name="text">
           <string>n Ascii Files (one for Each Reduced Data Set)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="base_name_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Base Filenames:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="prefix_name_value">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>REF_L</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="run_name_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>_###_</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="suffix_name_value">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>reduced_data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ext_name_label">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>.txt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createAsciiButton">
       <property name="text">
        <string>Create Ascii File ...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="folder_error">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>CHECK FOLDER PERMISSION !</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>createAsciiButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>create_reduce_ascii_button_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>445</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>one_ascii_format</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>output_format_radio_buttons_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>n_ascii_format</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>output_format_radio_buttons_event()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>auto_qmin_button</sender>
   <signal>clicked(bool)</signal>
   <receiver>Dialog</receiver>
   <slot>auto_qmin_button()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>create_reduce_ascii_button_event()</slot>
  <slot>output_format_radio_buttons_event()</slot>
  <slot>auto_qmin_button()</slot>
 </slots>
</ui>
